pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -2077,9 +2113,9 @@ func (cc *XController) manageQueueJob(ctx context.Context, qj *arbv1.AppWrapper,
 					_, err00 := cc.genericresources.SyncQueueJob(qj, &ar)
 					if err00 != nil {
 						if apierrors.IsInvalid(err00) {
-							klog.Warningf("[manageQueueJob] Invalid generic item sent for dispatching by app wrapper='%s/%s' err=%v", qj.Namespace, qj.Name, err00)
+							klog.Warningf("[manageQueueJob] Invalid generic item sent for dispatching by app wrapper='%s/%s' - error: %v", qj.Namespace, qj.Name, err00)
```

Comment:
This should probably be an Error log.

Reply:
Here I just updated the formatting so I'm not sure about changing this. @z103cb added this a few months ago. Maybe there were reasons for using `klog.Warningf` here rather than Errorf.

Reply:
If I recall correctly, this was done to not generate a spurious error message when there's a bad definition for the generic resource. As they say beauty in in the eye of the beholder, if you will if this warning should be an error I would not object.